docs/en/latest/admin-api.md

```diff
@@ -74,7 +74,7 @@ Note: When the `Admin API` is enabled, it will occupy the API prefixed with `/ap
 | desc             | False                                    | Auxiliary   | route description, usage scenarios, and more.                                                                                                                                                                                                                                                                                                                                                                                      | customer xxxx                                        |
 | uri              | True, can't be used with `uris`          | Match Rules | In addition to full matching such as `/foo/bar`、`/foo/gloo`, using different [Router](architecture-design/router.md) allows more advanced matching, see [Router](architecture-design/router.md) for more.                                                                                                                                                                                                                         | "/hello"                                             |
 | uris             | True, can't be used with `uri`           | Match Rules | The `uri` in the form of a non-empty list means that multiple different uris are allowed, and match any one of them.                                                                                                                                                                                                                                                                                                               | ["/hello", "/word"]                                  |
-| host             | False, can't be used with `hosts`        | Match Rules | Currently requesting a domain name, such as `foo.com`; PAN domain names such as `*.foo.com` are also supported.                                                                                                                                                                                                                                                                                                                    | "foo.com"                                            |
+| host             | False, can't be used with `hosts`        | Match Rules | Currently requesting a domain name or IP address, such as `foo.com`, `127.0.0.1` ; PAN domain names such as `*.foo.com` are also supported, `*.0.0.1` also.                                                                                                                                                                                                                                                                                                                    | "foo.com"                                            |
```

Comment:
CIDR?

Reply:
Not CIDR.
CIDR is like: "a.b.c.0/24"
But this is like: "*.0.0.1"
😀

Reply:
Better not mention this "feature" as it is supported unexpectedly. It just happens to work.

Reply:
OK.Got it.